flow/designs/sky130hd/aes/config.mk

```diff
@@ -4,7 +4,6 @@ export PLATFORM    = sky130hd
 
 export VERILOG_FILES = $(sort $(wildcard ./designs/src/$(DESIGN_NICKNAME)/*.v))
 export SDC_FILE      = ./designs/$(PLATFORM)/$(DESIGN_NICKNAME)/constraint.sdc
-export RCX_RULES     = $(PLATFORM_DIR)/rcx_patterns.rules
```

Comment:
Thanks, I have no idea why this is being set in the design config.

Reply:
RCX was disabled globally on the platform config for SKY130 and enabled only for a couple of designs, the goal of the PR is to enable it globally for SKY130.